packages/shell-api/src/shell-api.ts

```diff
@@ -0,0 +1,133 @@
+import AsyncWriter from '@mongosh/async-rewriter';
+import Mapper from '@mongosh/mapper';
+import { types, Help } from '@mongosh/shell-api-types';
+
+class ShellApi {
+  public mapper: any;
+  private asyncWriter: AsyncWriter;
+  private bus: any;
+  private container: any;
+  constructor(serviceProvider, bus, container) {
```

Comment:
Passing a reference to the container so that we can start/stop telemetry: yes or no?